Core/TransactionHandler.php

```diff
@@ -234,10 +234,10 @@ public function getTransactionMaxAmount($sTransactionId)
         $aResult = $oDb->select($sDbQuery, $aQueryArgs);
 
         if ($aResult !== false && $aResult->count() > 0) {
-            $fChildAmount = $aResult->fields['childTransactionsTotalAmount'];
+            $fChildAmount = (float) $aResult->fields['childTransactionsTotalAmount'];
         }
 
-        return $fBaseAmount - $fChildAmount;
+        return round(bcsub($fBaseAmount, $fChildAmount, Helper::BCSUB_SCALE), Helper::ROUND_PRECISION);
```

Comment:
Would it make sense to use the `Currency` model and use the precision set by the merchant for `ROUND_PRECISION`?

Reply:
👍 

Now the precision set by the merchant for this currency is used and the constant value here is only used as a fallback.